Finished_Pluggins/edit_shifts.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/* 
+Plugin Name: edit_shifts
+Description: Adds two forms to a page to be used by security to create a new shift or remove an existing
+one through the use of dropdown menus.
+Author: Jacob Tower
+*/
+
+//called by the shortcode - creates/displays forms to both add and remove shifts
+function createForms(){
+	global $wpdb;
+	
+	//call form creation methods and display forms on screen
+	$newShift = newForm();
+	echo $newShift . "</br>";
+	$removeShift = removeForm();
+	echo $removeShift;
+	
+	//runs if submit button on createShift form is pressed
+    if(isset($_POST['submitNew']) ){	
+        //store information from each form
+  		$driver = $_POST['formDriver'];
+  		$vehicle = $_POST['formVehicle'];
+  		$dateY = $_POST['formYear'];
+  		$dateM = $_POST['formMonth'];
+  		$dateD = $_POST['formDay'];
+  		$my_date = $dateY . '-' . $dateM . '-' . $dateD;
+  		$dayOfWeek = $_POST['formDOW'];
+  		$start = $my_date . ' ' . $_POST['formStartTime'] . ':00';
+  		$end = $my_date . ' ' . $_POST['formEndTime'] . ':00';
+  		  		
+		//create array of values to input to DB
+		$insert = array('driver_ID' => $driver, 
+						'vehicle_ID' => $vehicle, 
+						'date' => $my_date, 
+						'dayOfWeek' => $dayOfWeek, 
+						'startTime' => $start, 
+						'endTime' => $end);
+		
+  		$wpdb->insert('shifts', $insert);	//insert new row into DB
+	}
+	
+	
+	//runs if submit button on removeShift form is pressed
+	if(isset($_POST['submitRemove'])){
+		global $wpdb;
+		
+		$wpdb->delete('ride_requests', array('shift_ID' => $_POST['formShift']));	//must delete child rows first
+		
+		$remove = array('id' => $_POST['formShift']);	//array of values to remove in column => value form
+		echo "rows deleted: " .  $wpdb->delete('shifts', $remove);	//remove selected row from DB
```

Comment:
Either remove this line with the echo statement, or fix it so that it says something better - users will see these debugging comments.